react/components/UI/CookieLinkUnlessCurrent/index.js

```diff
@@ -0,0 +1,51 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import Cookies from 'cookies-js';
+import styled from 'styled-components';
+import { NavLink } from 'react-router-dom';
+
+const Link = styled(NavLink)`
+  display: block;
+  color: ${x => x.theme.colors.gray.regular};
+
+  ${({ isActive, theme }) => isActive && isActive() && `
+    color: ${theme.colors.gray.semiBold};
+    cursor: default;
+  `}
+`;
```

Comment:
I think having this styling here for something this abstract is a problem. And also if you're abstracting some pattern somewhere, use it where you found the original.